src/app/create-report/create-report.component.ts

```diff
@@ -252,11 +255,13 @@ export class CreateReportComponent implements OnInit, AfterViewInit {
   pageSize = PAGE_SIZE;
 
   dateDropdownOptions: DateFilterDropdownOption[] = [
-    { displayText: DateFilterName.YESTERDAY, numDays: 1 },
+    { displayText: DateFilterName.YESTERDAY, numDays: 1 }, 
     { displayText: DateFilterName.LAST_TWO_DAYS, numDays: 2 },
+    { displayText: DateFilterName.LAST_THREE_DAYS, numDays: 3 },
+    { displayText: DateFilterName.LAST_FOUR_DAYS, numDays: 4 },
+    { displayText: DateFilterName.LAST_FIVE_DAYS, numDays: 5 },
+    { displayText: DateFilterName.LAST_SIX_DAYS, numDays: 6 },
```

Comment:
Do we need these?

Reply:
@kilemensi I wanted to give all the day options since it is now down to only 7 days. What do you suggest would be the best filtering options

Reply:
Not sure @kelvinkipruto ... 1 day or 1 week (7 days) is good enough for me. I don't see myself picking 4 days or 6... just too weird.

Does the date picker still work?

Reply:
@kilemensi The datepicker does not work anymore.